### ISSUE TITLE
BUG: Logging when gridsearching is not reflected in yaml files.

### ISSUE THREAD

**Issue by Valleo4** [andersbogsnes/ml_tooling#316]
using the boston dataset from the boston data notebook and the same linear model

`with linear.log("./bostondata_linear"):
    linear.gridsearch(bostondata, { "normalize": [False, True] })`

yields incorrect yaml files with estimator: params: normalize: True for all yaml logs
and produces 4 files.

example:
```yaml
created_time: 2019-10-31 17:35:14.924329
estimator:
- classname: LinearRegression
  module: sklearn.linear_model.base
  params:
    copy_X: true
    fit_intercept: true
    n_jobs: null
    normalize: true
estimator_path: null
git_hash: afa6def92a1e8a0ac571bec254129818bb337c49
metrics:
  r2: 0.7160133196648374
model_name: BostonData_LinearRegression
versions:
  ml_tooling: 0.9.1
  pandas: 0.25.2
  sklearn: 0.21.3
```

**Comment by andersbogsnes** [comment]
closed by #343
